curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443cc3ed4a11e02329c4c0e.md

```diff
@@ -0,0 +1,180 @@
+---
+id: 6443cc3ed4a11e02329c4c0e
+title: Step 48
+challengeType: 0
+dashedName: step-48
+---
+
+# --description--
+
+`Math.floor` is a built-in method that rounds a number down to the nearest integer. For example, `Math.floor(3.9)` returns `3`. You will declare a constant named `minutesAgo` and assign it `Math.floor(timeDifference / MS_PER_MINUTE)` to get the number of minutes ago the post was created. The calculations are done by dividing the time difference by the number of milliseconds in a minute and then rounding down to the nearest integer.
+
+# --hints--
+
+You should declare a constant named `minutesAgo` and assign it `Math.floor(timeDifference / MS_PER_MINUTE)`.
```

Comment:
I think, `minutesAgo` should be in all caps as it is a constant.

Reply:
That's not really a convention we've been following in the projects - if we want to start, we should discuss what constants are to be all caps and what are to be camelCase.